optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -46,13 +50,21 @@
 @XStreamAlias("singleBenchmarkResult")
 public class SingleBenchmarkResult implements BenchmarkResult {
 
-    private static final Logger logger = LoggerFactory.getLogger(SingleBenchmarkResult.class);
+    // required by JAXB to refer to existing instances of this class
```

Comment:
(code style, very soft): the first letter after "// " should be a capital, so "// Required ...", even if it's not a sentance

Basically same principles as javadocs.
((If it's a sentence (subject + verb + ...), it must end with a ".", otherwise it shouldn't. This is already correct.))